source/agora/test/Base.d

```diff
@@ -937,35 +940,47 @@ public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)
     NodeConfig[] node_configs;
     Config[] configs;
 
+    void addAdditionalValidator()
+    {
+        auto validator_count = node_configs.filter!(conf => conf.is_validator).array.length;
+        foreach (idx; validator_count .. ValidateCountInGenesis)
+            node_configs ~= makeNodeConfig(true);
+    }
+
```

Comment:
I'm not sure I follow... Doesn't this add extra validators to the topology ? So if the user specifies e.g. 4 validators, he'll get 4 validators + whatever are in the genesis block ?

Reply:
https://github.com/bpfkorea/agora/pull/886#pullrequestreview-420698403
I am implementing what you have proposed, but I think I understand it differently.
I understood like this, can you check if it's right?
1. There is the enrollment of a fixed number of validators on the genesis block.
2. If the number of validators in the test configuration is greater than the fixed number, create additional validators.
3. When additional validators are created, they will eventually register themselves.

Reply:
And I only worked on 'agora.test.EnrollmentManager' and I haven't finished another unit test yet.
I wanted to know if what you've done so far is the same as what you think.